### ISSUE TITLE
ogr2ogr relation already exists error

### ISSUE THREAD

**Issue by pnoll1** [pnoll1/osmand_map_creation#59]
oa_qa branch is having issues with ogr2ogr call. First call after running with master branch runs fine. Subsequent runs fail with output below:
```
pat@desktop:~/projects/osmand_map_creation/osmand_osm/osm$ ogr2ogr PG:dbname=gis us/ri/statewide-addresses-state.geojson -nln us_ri_statewide_addresses_state_temp -overwrite -lco OVERWRITE=YES
ERROR 1: ERROR:  relation "us_ri_statewide_addresses_state_temp_wkb_geometry_geom_idx" already exists

ERROR 1: CREATE INDEX failed for layer us_ri_statewide_addresses_state_temp.
ERROR 1: Unable to write feature 0 from layer statewide-addresses-state.
ERROR 1: Terminating translation prematurely after failed
```

**Comment by pnoll1** [comment]
Using _temporary instead of _temp solves the issue. Ogr2ogr must be using _temp, but I can't find anything in the gdal docs.

Need to regenerate pg dumps and redo tests for name change.

**Comment by pnoll1** [comment]
Has OA implemented this protection already?

https://github.com/openaddresses/batch/blob/master/task/lib/job.js check_stats()

**Comment by pnoll1** [comment]
changing away from _temp does not solve issue. Will work on getting rid of ogr2ogr and doing sql directly.  Alternative is using ogr2ogr, but getting rid of spatial index which is geom_idx.

**Comment by pnoll1** [comment]
fixed in 85c9263cc2fe02ade50975f0e1fc59ede6974fdb. Drop index to make sure ogr2ogr can insert properly into table
